### PR TITLE
Inverse of the Log Minus Digamma function

### PR DESCRIPTION
**Pull request by 9il**
~~Depends on #3021.~~
See also Log Minus Digamma function #2957.

### CODE REVIEW

**Comment by burner** [comment]
could you add a See_Also to both functions so they are easier to find.

**Comment by 9il** [comment]
OK

**Comment by 9il** [comment]
Added.

**Comment by burner** [comment]
LGTM

**Comment by burner** [comment]
Auto-merge toggled on

**Comment by 9il** [comment]
Thanks!

**Comment by burner** [comment]
thank you

File: std/internal/math/gammafunction.d [hunk 1]
```diff
@@ -1571,3 +1571,64 @@ unittest {
...
+ *      Handbook of mathematical functions. Dover, New York,
+ *      pages 258-259, equation 6.3.18.
+ * 
+ * Authors: Ilya Yaroshenko
+ */
+real logmdigammaInverse(real y)
+{
+    import std.numeric: findRoot;
+    enum maxY = logmdigamma(real.min_normal);
+    static assert(maxY > 0 && maxY <= real.max);
```
Comment (ibuclaw) [inline std/internal/math/gammafunction.d]: This enum does not work for non-x86 targets (and gdc, which doesn't provide y2lx at compile-time).

Comment (ibuclaw) [inline std/internal/math/gammafunction.d]: https://github.com/D-Programming-Language/phobos/pull/3309

**Pull request merged**